app/services/info_store.rb

```diff
@@ -152,15 +152,15 @@ def median_note_length
     query = <<-SQL
       SELECT
         percentile_disc(0.5)
-        WITHIN group (order by note_size)
+        WITHIN group (order by note_size.column_size)
         FROM
           (SELECT
-             pg_column_size(annotation) 
+             pg_column_size(annotation) as column_size
              FROM highlights
              WHERE annotation IS NOT NULL) as note_size
     SQL
 
-    ActiveRecord::Base.connection.select_value(query)&.gsub(/[()]/, "").to_i
+    ActiveRecord::Base.connection.select_value(query).to_i
```

Comment:
It looks like the old code expected this to potentially be nil, just double checking this is a safe cast.